WordPress/Classes/ViewRelated/Settings/AboutViewController.swift

```diff
@@ -82,14 +82,17 @@ public class AboutViewController : UITableViewController
         var cell = tableView.dequeueReusableCellWithIdentifier(reuseIdentifier) as? UITableViewCell
         if cell == nil {
             cell = WPTableViewCell(style: .Value1, reuseIdentifier: reuseIdentifier)
-            WPStyleGuide.configureTableViewCell(cell)
         }
         
         let row = rows[indexPath.section][indexPath.row]
         
         cell!.textLabel?.text       = row.title
         cell!.detailTextLabel?.text = row.details ?? String()
-        cell!.accessoryType         = (row.handler != nil) ? .DisclosureIndicator : .None
```

Comment:
Thanks for changing this, since (all of the) rows actually display a modal webView, this really makes sense!